generators/app/tasks/appSetup/drawerFeatureFiles.js

```diff
@@ -1,37 +1,46 @@
+const DRAWER_FEATURE_PATH = ['src', 'app', 'components', 'Drawer'];
+const DRAWER_MENU_FEATURE_PATH = ['src', 'app', 'components', 'Drawer', 'components', 'DrawerMenu'];
+const DRAWER_OVERLAY_FEATURE_PATH = ['src', 'app', 'components', 'Drawer', 'components', 'DrawerOverlay'];
+
 module.exports = function drawerFeatureFiles() {
-  // src/components/Drawer/DrawerContainer.js
+  // src/app/components/Drawer/index.js
   this.fs.copy(
-    this.templatePath('src', 'components', 'Drawer', 'DrawerContainer.js'),
-    this.destinationPath(this.projectName, 'src', 'components', 'Drawer', 'DrawerContainer.js')
+    this.templatePath(...DRAWER_FEATURE_PATH, 'index.js'),
+    this.destinationPath(this.projectName, ...DRAWER_FEATURE_PATH, 'index.js')
   );
-  // src/components/Drawer/DrawerMenu.js
+  // src/app/components/Drawer/components/DrawerMenu/index.js
   this.fs.copy(
-    this.templatePath('src', 'components', 'Drawer', 'DrawerMenu.js'),
-    this.destinationPath(this.projectName, 'src', 'components', 'Drawer', 'DrawerMenu.js')
+    this.templatePath(...DRAWER_MENU_FEATURE_PATH, 'index.js'),
```

Comment:
because you are changing a 
`Drawer/Drawer/DrawerMenu.js`  by 
`Drawer/components/DrawerMenu/index.js`
So container should be an `layout.js` file, right?